clients/client/src/upload.js

```diff
@@ -62,7 +62,7 @@ const upload = async ({
   const retryCfg = {
     retries: retries === undefined ? 5 : retries,
     delayFactor: delayFactor === undefined ? 100 : delayFactor,
-    randomizationFactor: randomizationFactor === undefined ? randomizationFactor : 0.25,
+    randomizationFactor: randomizationFactor === undefined ? 0.25 : randomizationFactor,
```

Comment:
for some reason it didn't cause any problem before node20 upgrade

Reply:
😬

Reply:
Nice catch!